hs_access_control/management/commands/permissions.py

```diff
@@ -0,0 +1,51 @@
+"""
+This prints the active permissions of an access control relationship between a user and a resource.
+This is invaluable for access control debugging.
+
+"""
+
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+from hs_core.hydroshare.utils import get_resource_by_shortkey
+from hs_access_control.models.utilities import coarse_permissions
+from hs_access_control.management import user_from_name
+
+
+def usage():
+    print("Permissions usage:")
+    print("  permissions {username} {resource-id}")
+    print("Where:")
+    print("  {username} is a user name.")
+    print("  {resource-id} is a 32-character resource guid.")
+
+
+class Command(BaseCommand):
+    help = """Print access control provenance."""
+
+    def add_arguments(self, parser):
+
+        # a command to execute
+        parser.add_argument('arguments', nargs='*', type=str)
```

Comment:
why not just use username and resource_id here? arguments is not descriptive.

Reply:
Done.